src/btree/bt_ret.c

```diff
@@ -186,23 +186,38 @@ __wt_read_cell_time_window(WT_CURSOR_BTREE *cbt, WT_TIME_WINDOW *tw)
     return (true);
 }
 
+/*
+ * __read_page_cell_data_ref_kv --
+ *     Helper function to copy the time window and set a buffer to reference the data from an
+ *     unpacked key value cell. Return WT_RESTART if the cell is an overflow removed value.
+ */
+inline static int
```

Comment:
This is usually `static inline` - I'm not sure if it matters, but I'd rather stick to the convention.